src/ripple/app/ledger/Ledger.cpp

```diff
@@ -600,13 +600,13 @@ Ledger::peek(Keylet const& k) const
 }
 
 hash_set<PublicKey>
-Ledger::negativeUnl() const
+Ledger::negativeUNL() const
```

Comment:
This could be addressed later, but I wonder whether this method should be named `disabledValidators()` now?

Reply:
I thought about it, but did not change it. My reasoning was that the same thing (the `negativeUNL`) was accessed here and used somewhere else. If I change here, I probably should change other places as well, which I felt confusing. I see the `disabledValidator` more as a type and `disabledValidators` as its plural (or container). And I see negativeUNL as a usage of the container. Iirc, the skip list has type hashes, but it is protocol meaning is the skip list.

Reply:
That's fine.  It's one that we can think about.  I've been thinking of the `NegativeUNL` as a heterogeneous container of three elements:

- DisabledValidators,
- ValidatorToDisable, and
- ValidatorToReEnable

This specific entry point returns only DisabledValidators.  That's why I think the rename is appropriate.  But this name doesn't leak out into the API.  So we can change it, or not, later.